### DISCUSSION TITLE
Stripe payment management

### DISCUSSION THREAD

**Discussion by LouisLouiseKay** [funmusicplace/mirlo#163] [Ideas]
Stripe payments presents varying challenges that require some thinking, here are some of points outlined by the community already : 

- Micropayment (around 1$) fees are very high.
- There are fixed costs on top of the Stripe "cut percentage", which means several small transactions are pricier than just a single one.
- Kat from Banchan created a spreadsheet for calculating fees (available here upon request https://docs.google.com/spreadsheets/d/1vKHO01KTQ2FLbNR7D2IE5tUS1TP8hOtqoCJ7pLLfq-o/edit) 
- Stripes fees are higher than Paypal's in some circumstances, but Paypal's API is extremely complex to implement so we focus on managing Stripe as best we can for now.

In light of the elements outlined above, there are several options considered, that come with their own pros and cons. Mainly, there are two different ways that Stripe accounts can be setup : 

- Online marketplaces (including businesses such as Deliveroo, ManoMano etc.) that collect payment from customers, and later pay out to sub-accounts or service providers.
- Platforms that onboard other businesses and enable them to accept payments directly, such as Shopify or Squarespace.

We currently operate with the second option. The advantages are : 

- Simplicity in terms of implementation.
- Simplicity in terms of process for the user ; we just process payments immediately and they receive the money.
- Legal/Accounting simplicity : we don't withhold money so there's much less to worry about in terms of managing the way we process payments.

The Downsides are : 

- It's not possible to use the new Stripe / Paypal integration in the countries where it's available (UE mostly) using that method.
- Fees are higher because every transaction is accounted for, which incidentally means some people might make less money than just using Bandcamp solely because of transaction fees.

The first other option's advantages are, pretty much the opposite : 

- Harder to implement.
- Creates legal and accounting complexity : Our structure would need to put in place a system to withhold payment for a certain amount of time so that multiple transactions from multiple users are "bundled" in a one time payment, thus reducing payment fees.
- It also adds complexity on the user's end for managing their own accounting. Which means we need to put a system in place (like Bandcamp does) for them to be able download their payments / orders / invoices / platform fees in the form of thourough Spreadsheets so that they can share with their accountant because the paypal/stripe payment will not match the transaction payments.

Finally, it still needs to be determined how much of a cut would be acceptable from Mirlo to take as a platform, knowing we would like to at least not take more than what Bandcamp has setup as the default "standard" for the independant music economy, and possibly less if we can still operate the platform comfortably that way or find other stable sources of income.

**Comment by meljoann** [comment]
Depending if it's sustainable for you, could compete with:
- itch.io and their opt-in / user-definable platform cut 
- ko-fi and their system of "tips are free, subscription we take a cut"

**Reply by LouisLouiseKay** [reply]
Can you decide for the platform to have no cut at all on itch?

I like the ko-fi approach and I like the idea there could be a tipping option that's neither subscription or purchase.

**Reply by meljoann** [reply]
Yeah, it's like an honour system... I guess good for people starting out too, if you don't know the platform yet

**Comment by simonv3** [comment]
This has been implemented. A next feature request is adding paypal

**Comment by nemobis** [comment]
Did the PayPal vs Stripe comparison include different jurisdictions and payment sources? In the euro area, with payments backed by SEPA direct debit (SDD), the fees should be lower. Liberapay ended up using only Stripe for similar reasons.
